### PR TITLE
Add port flags to different dashboard cmds

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**
Allow users to specify component ports.
And also fix default skywalking port.

### CODE REVIEW

File: istioctl/cmd/dashboard.go [hunk 1]
```diff
@@ -510,17 +525,35 @@ func dashboard() *cobra.Command {
...
+	kiali := kialiDashCmd()
+	kiali.PersistentFlags().IntVar(&kialiPort, "kiali_port", defaultKialiPort, "Kiali port")
+	dashboardCmd.AddCommand(kiali)
+
+	prom := promDashCmd()
+	prom.PersistentFlags().IntVar(&promPort, "prom_port", defaultPrometheusPort, "Prometheus port")
+	dashboardCmd.AddCommand(prom)
+
+	graf := grafanaDashCmd()
+	graf.PersistentFlags().IntVar(&grafanaPort, "grafana_port", defaultGrafanaPort, "Grafana port")
```
Comment (howardjohn) [inline istioctl/cmd/dashboard.go:537]:
```suggestion
	graf.PersistentFlags().IntVar(&grafanaPort, "grafana-port", defaultGrafanaPort, "Grafana port")
```

I would just have a single `--port` one though, we already know if its "grafana" or "kiali" based on the dashboard command

Comment (hanxiaop) [reply istioctl/cmd/dashboard.go]: `--port` has already been used for the local port to listen to. I just uniformed to a `--ui-port` flag for each command.

**Review by GregHanson — APPROVED** [review]
release note looks good

**Comment by hanxiaop** [comment]
not stale

**Comment by ericvn** [comment]
Need to do a little testing, but I notice the skywalking istio.io test timed out when it was run against a build containing this commit. I am trying again with just this commit in https://github.com/istio/istio.io/pull/12962. The prior run which ran was run against the commit just before this one.

**Comment by hanxiaop** [comment]
> Need to do a little testing, but I notice the skywalking istio.io test timed out when it was run against a build containing this commit. I am trying again with just this commit in [istio/istio.io#12962](https://github.com/istio/istio.io/pull/12962). The prior run which ran was run against the commit just before this one.

Sorry didn't notice this was merged. I think the default port is not correct, I'll revise to the correct one.

**Pull request merged**